runtime/queries/cylc/injections.scm

```diff
@@ -0,0 +1,18 @@
+((setting
+  key: (key) @key (#match? @key "^script$|\-script$|^script\-")
```

Comment:
I believe you need double backslashes here since the first is escaped:

```suggestion
  key: (key) @key (#match? @key "^script$|\\-script$|^script\\-")
```

but also I think it shouldn't be necessary to escape the `-`. IIRC it needs to be escaped when used in the middle of a character class `[a\\-z]` but not otherwise. I haven't verified this though so it's worth checking if I'm correct here

Reply:
I chose to remove the escaping entirely. As you said, it was unecessary.